README.MacOSX

```diff
@@ -19,12 +19,12 @@ sudo xcode-select --install
 sudo xcode-select -switch /Applications/Xcode.app/Contents/Developer
 
 For Homebrew:
-brew install cmake readline icu4c openssl
+brew install openssl readline curl icu4c libiconv zlib cmake python3
 
 For MacPorts:
-port install autoconf binutils cmake ncurses zlib readline
+port install openssl binutils cmake zlib readline python37
```

Comment:
If you're going to remove autotools (autoconf automake libtool) from instructions, you may want to do it in all of them.

Reply:
updated a README in static-build dir:
```
--- a/static-build/README.md
+++ b/static-build/README.md
@@ -21,7 +21,6 @@ be necessary for building and testing:
   intend to use the <E2><80><9C>Run the test suite". For all platforms, this is python
   version 3.x. You need the following Python modules:
   * pyyaml
-  * argparse
   * msgpack-python
   * gevent
   * six
@@ -33,7 +32,7 @@ CentOS:
 ```bash
 yum install -y \
     git perl gcc cmake make gcc-c++ libstdc++-static autoconf automake libtool \
-    python3-msgpack python3-yaml python-argparse python3-six python3-gevent
+    python3-msgpack python3-yaml python3-six python3-gevent
 ```
 
 Ubuntu/Debian:
@@ -41,8 +40,7 @@ Ubuntu/Debian:
 ```bash
 apt-get install -y \
     build-essential cmake make coreutils autoconf automake libtool sed \
-    python3 python3-pip python3-setuptools python3-dev \
-    python3-msgpack python3-yaml python-argparse python3-six python3-gevent
+    python3 python3-msgpack python3-yaml python3-six python3-gevent
 ```
 
 MacOS:
```